### PR TITLE
Update node canary

### PR DESCRIPTION
**Pull request by sbc100**
This involved using toIndexType in a few more places due to recent wasm64 changes requiring BitInt in more places.

Split out from #22773

### CODE REVIEW

**Review by dschuff — APPROVED** [review]

File: test/core/test_module_wasm_memory.js [hunk 1]
```diff
@@ -6,8 +6,11 @@
...
  */
 
-Module['wasmMemory'] = new WebAssembly.Memory({ 'initial': 256, 'maximum': 256,
+Module['wasmMemory'] = new WebAssembly.Memory({
+  'initial': {{{ toIndexType('256') }}},
+  'maximum': {{{ toIndexType('256') }}},
 #if MEMORY64
-  'index': 'i64'
+  'index': 'i64',
+  'address': 'i64',
```
Comment (kripken) [inline test/core/test_module_wasm_memory.js:14]: Out of curiosity, what is the difference in the spec between these two?

**Review by kripken — APPROVED** [review]

File: test/core/test_module_wasm_memory.js (hunk 1 above)
Comment (sbc100) [reply test/core/test_module_wasm_memory.js:14]: Added a comment.

**Review by kripken — APPROVED** [review]

**Comment by dschuff** [comment]
This seems to have broken the wasm64l tests on Chromium CI: https://ci.chromium.org/ui/p/emscripten-releases/builders/try/linux/b8732162502277725409/overview
Do we need to require node canary for those now too?

**Comment by dschuff** [comment]
Actually no, of course we shouldn't; the whole point of wasm64l is to work on MVP...

**Comment by sbc100** [comment]
Fix is in https://github.com/emscripten-core/emscripten/pull/22853

**Comment by dschuff** [comment]
Still seems broken? See https://chromium-review.googlesource.com/c/emscripten-releases/+/5995675

**Pull request merged**